.dockerignore

```diff
@@ -0,0 +1,2 @@
+/target
+**/*.rs.bk
```

Comment:
```suggestion

```

I would remove the `*.rs.bk` file. I don't know what generates it, but i would create a global [gitignore](https://git-scm.com/docs/gitignore) file:

> Patterns which a user wants Git to ignore in all situations (e.g., backup or temporary files generated by the user’s editor of choice) generally go into a file specified by core.excludesFile in the user’s ~/.gitconfig. Its default value is $XDG_CONFIG_HOME/git/ignore. If $XDG_CONFIG_HOME is either not set or empty, $HOME/.config/git/ignore is used instead.